spring-amqp/src/main/java/org/springframework/amqp/core/Address.java

```diff
@@ -39,14 +39,25 @@
  */
 public class Address {
 
+	/**
+	 * Use this value in {@code RabbitTemplate#setReplyAddress(String)} to use a temporary reply queue
+	 * for each request, regardless of whether the broker supports direct reply-to.
+	 */
+	public static final String TEMPORARY_REPLY_QUEUE_TOKEN = "##TEMPORARY_QUEUE##";
```

Comment:
Null won't work from XML.

OK, enough back pressure - :smile: - I'll make it a boolean (with namespace support in 1.6 only).